walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,67 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body {
+  font-family: Arial, sans-serif;
+  padding: 4px;
+  text-align: center;
 }
 
-.image-thumbnails {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
+h1 {
+  text-align: center;
+  font-weight: normal;
+}
+
+nav {
+  background: white;
+  width: 100%;
+  margin-bottom: 20px;
+  position: fixed;
+  top: 0;
+  padding-top: 1vw;
+  padding-bottom: 1vw;
+}
+
+nav ul {
+  margin: 0;
+  padding: 0;
+}
+
+nav ul li {
+  list-style: none;
+  text-align: center;
+  font-size: calc(14px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+  line-height: calc(1.3em + (1.5 - 1.2) * ((100vw - 300px)/(1600 - 300)));
+}
+
+nav ul li a {
+  display: block;
+  text-decoration: none;
+  color: black;
+  padding: 10px 0;
+  
 }
 
-.image-thumbnails img {
-  margin: 10px;
-  max-height: 200px;
-  max-width: 200px;
+nav {
+  display: table;
+  table-layout: fixed;
 }
 
-.image-thumbnails img:hover {
-  border: thick solid black;
+ul {
+  display: flex;
+  flex-direction: row;
 }
 
-#random-image-container img {
-  margin-top: 20px;
+ul li {
+    flex-grow: 1;
 }
```

Comment:
This could be defined in the `nav ul li` rule above. 

Also, it's preferable to just use `flex: 1;` here as it will intelligently set the other values for `flex-shrink` and `flex-basis`. Also recommended by the styleguide to use shorthand properties whenever possible: https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties